testing/devnet/.env.example

```
@@ -19,3 +19,11 @@ NODE4_OMNI_PORT=53564
 
 # Persistence mode (set to 1 for persistent volumes)
 PERSISTENT=0
+
+# Petri Consensus (enabled by default; set to false to fall back to PoRBFT v2)
+PETRI_CONSENSUS=true
+PETRI_FORGE_INTERVAL_MS=2000
+PETRI_BLOCK_INTERVAL_MS=10000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PETRI_BLOCK_INTERVAL_MS key should go before the PETRI_CONSENSUS key

(UnorderedKey)